emsdk.py

```diff
@@ -2775,7 +2770,14 @@ def extract_bool_arg(name):
 
   arg_old = extract_bool_arg('--old')
   arg_uses = extract_bool_arg('--uses')
+  arg_permanent = extract_bool_arg('--permanent')
```

Comment:
Could you use persistent as discussed in the other bug?  I know that permanent and persistent are essentially synonyms but my sense is that persistent is more commonly used the tech world.

Reply:
That is the term that has been used in the readme: https://github.com/emscripten-core/emsdk#how-do-i-change-the-currently-active-sdk-version

I don't mind changing it, but I found permanent more natural.